runner/src/internal.rs

```diff
@@ -54,16 +54,25 @@ pub struct RunExamples {
     pub example_name: Option<String>,
     #[structopt(long, help = "only build the examples, do not run them")]
     pub build_only: bool,
+    #[structopt(flatten)]
+    pub build_server: BuildServer,
 }
 
 #[derive(StructOpt, Clone)]
 pub struct BuildServer {
+    #[structopt(long, help = "server variant: [base]", default_value = "base")]
+    pub server_variant: String,
```

Comment:
Does this mean that `runner` can't do a non-`oak_debug` ("logless") build any more?

Reply:
As you noted below, it was not doing it already, I just updated the text.